PyTorch Sentiment Analysis Exercise.ipynb

```diff
@@ -301,9 +301,7 @@
     "        super(SentimentAnalyzer, self).__init__()\n",
     "        self.hidden_size = hidden_size\n",
     "        \n",
-    "        self.embedding = nn.Embedding(vocab_size, embedding_size)\n",
-    "        self.embedding.weight.data.copy_(pretrained_embedding)\n",
-    "        self.embedding.weight.requires_grad = False\n",
+    "        self.embedding = nn.Embedding.from_pretrained(pretrained_embedding, freeze=True)\n",
```

Comment:
Now that I've changed this to use `from_pretrained`, we don't need `vocab_size` in this class anymore. Feel free to remove it if you prefer to keep this change, or let me know if you want me to change this